packages/tasks/src/model-libraries.ts

```diff
@@ -241,6 +241,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://huggingface.co/docs/hub/keras",
 		snippets: snippets.keras,
 		filter: true,
+		countDownloads: `path:"config.json" OR path_extension:"keras"`,
+	},
+	"tf-keras": {
+		// Legacy "Keras 2" library (tensorflow-only)
+		prettyLabel: "TF-Keras",
+		repoName: "TF-Keras",
+		repoUrl: "https://github.com/keras-team/tf-keras",
```

Comment:
you will add docsUrl later?

Reply:
They don't really have a docs URL for `tf-keras`. I think it's best to redirect to github url and then people found what they went/need. I don't expect much traffic on this URL anyway.

Reply:
no i meant `docsUrl ` i.E. our docs https://huggingface.co/docs/hub/keras renamed to https://huggingface.co/docs/hub/tf-keras

as discussed yesterday :)

Reply:
done in https://github.com/huggingface/huggingface.js/pull/774/commits/2a72cc212395ce0c5d46ebce48b17720fc146b8e